src/data/set/finite.lean

```diff
@@ -785,6 +785,18 @@ finite.induction_on H
   (by simp only [bUnion_empty, bdd_above_empty, ball_empty_iff])
   (λ a s ha _ hs, by simp only [bUnion_insert, ball_insert_iff, bdd_above_union, hs])
 
+protected lemma finite.bdd_above_image {s : set β} (f : β → α) (h : s.finite) :
+  bdd_above (f '' s) :=
+begin
+  rcases is_empty_or_nonempty β with _i | _i; resetI,
+  { inhabit α,
+    use default α,
+    rintros a ⟨b, hb, rfl⟩,
+    revert hb b,
+    exact _i.elim },
+  exact (h.image f).bdd_above,
+end
```

Comment:
I think this is just:
```suggestion
(h.image f).bdd_above
```
which actually makes me wonder if we really need this lemma.

Reply:
What happens if you try to inline this term in your use case?

Reply:
Great, yes!  I'll abandon these lemmas.